File: Pembelian_tiket_pesawat.py
print('Nama :Feruziakinta Syakira')
print('Nim  :2310431008')
print('Tugas:Modul II')
print(' ')

print('Pembelian tiket pesawat Eagle')
print('Lokasi Awal : Padang')
print(' ')
print('SELAMAT DATANG')
print('SILAHKAN ISI')
print('')

nama=(input('Nama          :'))
umur=(input('Umur          :'))
JK=(input('Jenis kelamin :'))
print(' ')

print('|========================|')
print('|  Tujuan keberangkatan  |')
print('|1. Jakarta              |')
print('|2. Yogyakarta           |')
print('|3. Bandung              |')
print('|========================|')
print(' ')

pilih_tujuan=int(input('Pilih tujuan : '))
if(pilih_tujuan==1):
	print('Jakarta')
	Tujuan='Jakarta'
if(pilih_tujuan==2):
	print('Yogyakarta')
	Tujuan='Yogyakarta'
if(pilih_tujuan==3):
	print('Bandung')
	Tujuan='Bandung'
print(' ')

print('|=====================|')
print('|Jenis Maskapai       |')
print('|1. Ekonomi           |')
print('|2. Bussines          |')
print('|3. First Class       |')
print('|=====================|')
print(' ')

pilih_maskapai=int(input('Pilih Maskapai : '))
if(pilih_maskapai==1):
	print('Ekonomi')
	Maskapai='Ekonomi'  
if(pilih_maskapai==2):
	print('Business')
	Maskapai='Business'
if(pilih_maskapai==3):
	print('First Class')
	Maskapai='First Class'
print(' ')

if(pilih_tujuan==1) & (pilih_maskapai==1):
	Harga=1500000
if(pilih_tujuan==1) & (pilih_maskapai==2):
	Harga=2500000
if(pilih_tujuan==1) & (pilih_maskapai==3):
	Harga=3500000
if(pilih_tujuan==2) & (pilih_maskapai==1):
	Harga=1200000
if(pilih_tujuan==2) & (pilih_maskapai==2):
	Harga=2200000
if(pilih_tujuan==2) & (pilih_maskapai==3):
	Harga=3200000
if(pilih_tujuan==3) & (pilih_maskapai==1):
	Harga=1000000
if(pilih_tujuan==3) & (pilih_maskapai==2):
	Harga=2000000
if(pilih_tujuan==3) & (pilih_maskapai==3):
	Harga=3000000	

kursi=int(input('Jumlah kursi:'))
print(' ')
if (kursi>3):
	total_harga=int((Harga*kursi)-((Harga*kursi)*0.25))
	print('Diskon 25%')
if(kursi<=3):
	total_harga=(int(Harga*kursi))
print(' ')
print('Total Harga:',total_harga)
print(' ')

print('=====================================')
print('           Struk Pembelian            ')
print('Lokasi Awal   : Padang')
print('Nama          :',nama)		
print('Umur          :',umur)
print('Jenis Kelamin :',JK)
print('-------------------------------------')
print('Tujuan        :',Tujuan)
print('Maskapai      :',Maskapai)
print('Jumlah Kursi  :',kursi)
print('Total Harga   :',total_harga)
print('            TERIMA KASIH       ')
print('=====================================')

File: Pembelian_tiket_pesawat.pas
program pembelian_tiket_pesawat;
uses crt;
var
  nama,umur,JK,tujuan,maskapai:string;
  kursi,pilih_tujuan,pilih_maskapai:integer;
  harga,total_harga:real;
begin
  clrscr;
  writeln('Nama :Feruziakinta Syakira   |');
  writeln('NIM  :2310431008             |');
  writeln('Tugas:Modul II               |');
  writeln('___________|');
  writeln(' ');
  
  writeln('Pembelian tiket pesawat Eagle');
  writeln(' ');
  writeln('Lokasi Awal:Padang');
  writeln(' ');
  writeln('SELAMAT DATANG');
  writeln('SILAHKAN ISI');
  write('Nama           :');
  readln(nama);
  write('Umur           :');
  readln(umur);
  write('Jenis kelamin  :');
  readln(JK);
  writeln(' ');
  
  writeln('|====================|');
  writeln('|Tujuan keberangkatan|');
  writeln('|--------------------|');
  writeln('|1. Jakarta          |');
  writeln('|2. Yogyakarta       |');
  writeln('|3. Bandung          |');
  writeln('|====================|');
  writeln(' ');
  write('Pilih tujuan :');
  readln(pilih_tujuan);
  
  
  if (pilih_tujuan=1) then begin tujuan:='Jakarta';
  writeln('Jakarta');
  end; 
 
  if(pilih_tujuan=2) then begin tujuan:='Yogyakarta';
  writeln('Yogyakarta');
  end; 
 
  if(pilih_tujuan=3) then begin tujuan:='Bandung';
  writeln('Bandung');
  end;
  writeln(' ');
  
  writeln('|================|');
  writeln('| Jenis maskapai |');
  writeln('|1. Ekonomi      |');
  writeln('|2. Bussines     |');
  writeln('|3. First class  |');
  writeln('|================|');
  writeln(' ');
  write('Pilih maskapai :');
  readln(pilih_maskapai);
  
  if(pilih_maskapai=1) then begin maskapai:='Ekonomi';
  writeln('Ekonomi');
  end;
  
  if (pilih_maskapai=2) then begin maskapai:='Bussines';
  writeln('Business');
  end;
  
  if (pilih_maskapai=3) then begin maskapai:='First class';
  writeln('First class');
  end;
  writeln(' ');
  
  if (pilih_tujuan=1) and (pilih_maskapai=1) then begin harga:=1500000;
  end;
  
  if (pilih_tujuan=1) and (pilih_maskapai=2) then begin harga:=2500000;
  end;
  
  if (pilih_tujuan=1) and (pilih_maskapai=3) then begin harga:=3500000
  end;
  
  if (pilih_tujuan=2) and (pilih_maskapai=1) then begin harga:=1200000;
  end;
  
  if (pilih_tujuan=2) and (pilih_maskapai=2) then begin harga:=2200000;
  end;
  
  if (pilih_tujuan=2) and (pilih_maskapai=3) then begin harga:=3200000
  end;
  
  if (pilih_tujuan=3) and (pilih_maskapai=1) then begin harga:=1000000;
  end;
  
  if (pilih_tujuan=3) and (pilih_maskapai=2) then begin harga:=2000000;
  end;
  
  if (pilih_tujuan=3) and (pilih_maskapai=3) then begin harga:=3000000;
  end;
  
  write('Jumlah kursi:');
  readln(kursi);
  if (kursi>3) then begin total_harga:=((harga*kursi)-((harga*kursi)*0.25));
  writeln('Diskon 25%');
  end;
  if (kursi<=3) then begin total_harga:=(harga*kursi);
  end;
  writeln(' ');
  writeln('Total harga=',total_harga);
  writeln(' ');
  writeln('===================================');
  writeln('         Struk Pembelian           ');
  writeln('Lokasi Awal  : Padang              ');
  writeln('Nama         : ' ,nama              );
  writeln('Umur         : ',umur               );
  writeln('Jenis kelamin: ',JK                 );
  writeln('-----------------------------------');
  writeln('Tujuan       : ',Tujuan             );
  writeln('Maskapai     : ',Maskapai           );
  writeln('Jumlah kursi : ',Kursi              );
  writeln('Total harga  : ',total_harga        );
  writeln('           TERIMA KASIH            ');
  writeln('===================================');
  readln;
end.
